### PR TITLE
fix(curriculum): Convert blockquote elements to triple backtick syntax for JavaScript Algorithms and Data Structures

### PR DESCRIPTION
**Pull request by RandellDawson**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

**PR Purpose:** Converts `blockquote` elements to triple backtick syntax in the `Description` and `Instructions` sections of the `JavaScript Algorithms and Data Structures` challenges.

Related to issue #35991

### CODE REVIEW

**Review by moT01 — APPROVED** [review]
Looked at each of these locally - they all look good to me 👍 nice job @RandellDawson! 🎉

**Comment by moT01** [comment]
es6/understand-the-differences-between-import-and-require.english.md - I have a PR open on this file
es6/write-higher-order-arrow-functions.english.md - you have an open PR on this file

There's a third PR of mine that already got the backtick conversion - so there shouldn't be a problem there.

I'm not sure if you want to remove those files from this PR - or just merge it as is and fix any conflicts that might arise - I'm fine with merging this ahead of mine, I can fix conflicts there.

**Comment by RandellDawson** [comment]
I think we will just fix any that pop up.

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-equality-operator.english.md [hunk 1]
```diff
@@ -9,26 +9,10 @@ videoUrl: 'https://scrimba.com/c/cKyVMAL'
...
-```js
-function equalityTest(myVal) {
-  if (myVal == 10) {
-     return "Equal";
-  }
-  return "Not Equal";
-}
-```
-
+<blockquote>function equalityTest(myVal) {<br>&nbsp;&nbsp;if (myVal == 10) {<br>&nbsp;&nbsp;&nbsp;&nbsp; return "Equal";<br>&nbsp;&nbsp;}<br>&nbsp;&nbsp;return "Not Equal";<br>}</blockquote>
```
Comment (ojeytonwilliams) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-equality-operator.english.md]: Shouldn't this be js?

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-equality-operator.english.md [hunk 2]
```diff
@@ -9,26 +9,10 @@ videoUrl: 'https://scrimba.com/c/cKyVMAL'
...
 In order for JavaScript to compare two different <code>data types</code> (for example, <code>numbers</code> and <code>strings</code>), it must convert one type to another. This is known as "Type Coercion". Once it does, however, it can compare terms as follows:
-
-```js
-1   ==  1    // true
-1   ==  2    // false
-1   == '1'   // true
-"3"  ==  3    // true
-```
-
+<blockquote>1   ==  1    // true<br>1   ==  2    // false<br>1   == '1'   // true<br>"3"  ==  3    // true</blockquote>
```
Comment (ojeytonwilliams) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-equality-operator.english.md]: Shouldn't this be js?

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-inequality-operator.english.md [hunk 3]
```diff
@@ -9,15 +9,7 @@ videoUrl: 'https://scrimba.com/c/cdBm9Sr'
...
-
-```js
-1 != 2      // true
-1 != "1"    // false
-1 != '1'    // false
-1 != true   // false
-0 != false  // false
-```
-
+<blockquote>1 != 2      // true<br>1 != "1"    // false<br>1 != '1'    // false<br>1 != true   // false<br>0 != false  // false</blockquote>
```
Comment (ojeytonwilliams) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-inequality-operator.english.md]: Shouldn't this be js?

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-less-than-operator.english.md [hunk 4]
```diff
@@ -9,15 +9,7 @@ videoUrl: 'https://scrimba.com/c/cNVRWtB'
...
-
-```js
-2 < 5  // true
-'3' < 7  // true
-5 < 5  // false
-3 < 2  // false
-'8' < 4  // false
-```
-
+<blockquote>2 &lt; 5  // true<br>'3' &lt; 7  // true<br>5 &lt; 5  // false<br>3 &lt; 2  // false<br>'8' &lt; 4  // false</blockquote>
```
Comment (ojeytonwilliams) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-less-than-operator.english.md]: Shouldn't this be js?

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-less-than-or-equal-to-operator.english.md [hunk 5]
```diff
@@ -9,15 +9,7 @@ videoUrl: 'https://scrimba.com/c/cNVR7Am'
...
-
-```js
-4 <= 5  // true
-'7' <= 7  // true
-5 <= 5  // true
-3 <= 2  // false
-'8' <= 4  // false
-```
-
+<blockquote>4 &lt;= 5  // true<br>'7' &lt;= 7  // true<br>5 &lt;= 5  // true<br>3 &lt;= 2  // false<br>'8' &lt;= 4  // false</blockquote>
```
Comment (ojeytonwilliams) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-less-than-or-equal-to-operator.english.md]: Shoudn't this be js?

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-strict-equality-operator.english.md [hunk 6]
```diff
@@ -10,12 +10,7 @@ videoUrl: 'https://scrimba.com/c/cy87atr'
 Strict equality (<code>===</code>) is the counterpart to the equality operator (<code>==</code>). However, unlike the equality operator, which attempts to convert both values being compared to a common type, the strict equality operator does not perform a type conversion.
 If the values being compared have different types, they are considered unequal, and the strict equality operator will return false.
 <strong>Examples</strong>
-
-```js
-3 === 3   // true
-3 === '3' // false
-```
-
+<blockquote>3 === 3   // true<br>3 === '3' // false</blockquote>
```
Comment (ojeytonwilliams) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-strict-equality-operator.english.md]: Shouldn't this be js?

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-strict-inequality-operator.english.md [hunk 7]
```diff
@@ -9,13 +9,7 @@ videoUrl: 'https://scrimba.com/c/cKekkUy'
...
 The strict inequality operator (<code>!==</code>) is the logical opposite of the strict equality operator. It means "Strictly Not Equal" and returns <code>false</code> where strict equality would return <code>true</code> and <em>vice versa</em>. Strict inequality will not convert data types.
 <strong>Examples</strong>
-
-```js
-3 !== 3   // false
-3 !== '3' // true
-4 !== 3   // true
-```
-
+<blockquote>3 !== 3   // false<br>3 !== '3' // true<br>4 !== 3   // true</blockquote>
```
Comment (ojeytonwilliams) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-strict-inequality-operator.english.md]: Shouldn't this be js?

**Review by ojeytonwilliams — CHANGES_REQUESTED** [review]
There were a few cases where I thought some reverted blockquotes should have been kept as js.  I've checked everything locally, though, and it looks great otherwise!

**Comment by RandellDawson** [comment]
@ojeytonwilliams Regarding the several challenges you asked if they should be `j`, I originally I had them as code blocks, but @moT01 did a first pass (informal) review yesterday and thought they should be reverted back to blockquotes.  Looking at it again, I can see how they technically are JavaScript expressions.

**Comment by moT01** [comment]
There was a misunderstanding when I pointed out those comparison challenges yesterday @RandellDawson - I wasn't saying that they weren't `js` - what I meant was that they were misaligned and looked goofy

**Comment by ojeytonwilliams** [comment]
Fair enough.  @moT01 do you still prefer them as blockquotes?  I just figured they might as well be treated as code.

**Comment by moT01** [comment]
I think they work either way. I'll leave it up to Randell - if he wants to leave them, sure - or if he want's them changed back, he'll be the one to do it

**Comment by RandellDawson** [comment]
@ojeytonwilliams OK.  Hopefully this commit takes care of everything.  I made the 10 changes suggested above.

**Comment by RandellDawson** [comment]
@moT01 If it is all the same to you (which seems like it is), I have reverted back to the code blocks, since they really were JavaScript expressions.   I will also fixed the merge conflict on this, so we can get it merged soon.

**Comment by ojeytonwilliams** [comment]
I'm just checking this one final time.  Before I merge, there's potential conflict with three of your PRs, @moT01.  Shall we just merge this big one and deal with any conflicts in the smaller PRs?  Seems easier than repeatedly changing this one.  What do you think?

**Comment by moT01** [comment]
yes, that's fine

**Review by ojeytonwilliams — APPROVED** [review]

**Comment by ojeytonwilliams** [comment]
Okay, cool.  I went ahead and made some formatting changes.  I figured those were what you were talking about in the first place, @moT01. I've checked they all look fine locally, so if you're happy, please feel free to merge!

**Review by moT01 — APPROVED** [review]
🎉 🎉 🎉 🎉 🎉 🎉 🌮

**Pull request merged**